File: openresty/cdn-system/api/models/task.go
package models

import "time"

// Task 对应数据库中的 `task` 表
type Task struct {
	ID        int64     `json:"id" gorm:"primaryKey"`
	PID       int64     `json:"pid"`
	Pry       int       `json:"pry"`
	Name      string    `json:"name"`
	Type      string    `json:"type"` // refresh_url, refresh_dir, preheat
	Res       string    `json:"res"`
	Data      string    `json:"data"` // URLs
	Depend    string    `json:"depend"`
	CreateAt  time.Time `json:"create_at" gorm:"column:create_at"`
	StartAt   time.Time `json:"start_at" gorm:"column:start_at"`
	EndAt     time.Time `json:"end_at" gorm:"column:end_at"`
	Ret       string    `json:"ret"`
	Enable    bool      `json:"enable"`
	State     string    `json:"state"` // running, done, fail, waiting
	ErrTimes  int       `json:"err_times"`
	RetryAt   time.Time `json:"retry_at" gorm:"column:retry_at"`
	Progress  string    `json:"progress"`
}

func (Task) TableName() string {
	return "task"
}

File: cdn-system/cnn.net/src/Cnn.Common/Domain/Entities/Task.cs
using SqlSugar;

namespace Cnn.Domain.Entities;

[SugarTable("task")]
public class Task
{
    [SugarColumn(IsPrimaryKey = true, IsIdentity = true)]
    public long Id { get; set; }

    public int? Pid { get; set; }

    public int? Pry { get; set; }

    public string? Name { get; set; }

    public string? Type { get; set; }

    public string? Res { get; set; }

    public string? Data { get; set; }

    public string? TargetsJson { get; set; }

    public string? Depend { get; set; }

    public DateTime? CreateAt { get; set; }

    public DateTime? StartAt { get; set; }

    public DateTime? EndAt { get; set; }

    public string? Ret { get; set; }

    public bool? Enable { get; set; }

    public string? State { get; set; }

    public int? ErrTimes { get; set; }

    public DateTime? RetryAt { get; set; }

    public string? Progress { get; set; }
}
